core/src/main/java/org/mule/el/mvel/MVELExpressionLanguageWrapper.java

```diff
@@ -22,6 +23,14 @@ public MVELExpressionLanguageWrapper(MuleContext muleContext)
 
         if (muleContext.getExpressionManager() instanceof DefaultExpressionManager)
         {
+            try
```

Comment:
It's not. It's related to the comment on MVELExpressionLanguage.java. Will review as said before